File: dev/20190415/k2works/fizzbuzz/src/main/java/org/hiroshima_arc/domain/type/fizzbuzz/FizzBuzzType02.java
package org.hiroshima_arc.domain.type.fizzbuzz;

import org.hiroshima_arc.domain.model.fizzbuzz.FizzBuzzValue;

class FizzBuzzType02 extends IFizzBuzzType {
    @Override
    FizzBuzzValue generate(int number) {
        return new FizzBuzzValue(number,String.valueOf(number));
    }
}

File: dev/20190902/k2works/app/FizzBuzz/Domain/Type/FizzBuzzType02.cs
using FizzBuzz.Application.Service;
using FizzBuzz.Domain.Model;

namespace FizzBuzz.Domain.Type
{
    public class FizzBuzzType02: FizzBuzzType

    {
        public override IFizzBuzz Generate(int number)
    {
        return new FizzBuzzValue(number,number.ToString());
    }
    }
}